hardhat/deploymentInfo.json

```diff
@@ -121,11 +121,20 @@
       "address": "0xE6E340D132b5f46d1e472DebcD681B2aBc16e57E"
     },
     "CoSoul": {
-      "address": "0x82e01223d51Eb87e16A03E24687EDF0F294da6f1"
+      "address": "0x7969c5eD335650692Bc04293B07F5BF2e7A673C0"
```

Comment:
@CryptoGraffe was confused here for a bit, but seeing now that the proxy deploy creates deployments info for the Proxy, Implementation and the main contract (which is same as Proxy address) so we can now connect ignoring that proxy like: https://github.com/coordinape/coordinape/pull/2191/files#diff-b734c8ed9cd439ea59975ad4befcf67d6722e79bca44bbf0ea2105d2999f0b02R94